apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTimings.cs

```diff
@@ -61,8 +68,8 @@ internal RetryTimings Clone() => new()
     /// <summary>
     /// Returns the existing retry timings configurations updated with the specified initial backoff timings.
     /// </summary>
-    /// <param name="initialBackoff">Initial Backoff for retry</param>
-    /// <returns></returns>
+    /// <param name="initialBackoff">Initial Backoff for retry.</param>
+    /// <returns>Retry timings with updated initial backoff.</returns>
```

Comment:
Please see the pattern of documentation for methods starting with `With` and then update the documentation.